spec/requests/api/subscriptions/update_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe 'POST /api/subscriptions', type: :request do
+  let(:user) { create(:user) }
+  let(:credentials) { user.create_new_auth_token }
+
+  subject { response }
+
+  describe 'as a authenticated user, when subscribing' do
```

Comment:
Consider removing `, when subscribing` from this message. My point is that it is redundant.